src/mysql.c

```diff
@@ -685,6 +687,101 @@ static int mysql_read_innodb_stats (mysql_database_t *db, MYSQL *con)
 	return (0);
 }
 
+static int mysql_read_wsrep_stats (mysql_database_t *db, MYSQL *con)
+{
+	MYSQL_RES *res;
+	MYSQL_ROW  row;
+
+	const char *query;
+	struct {
+		const char *key;
+		const char *type;
+		int ds_type;
+	} metrics[] = {
+
+		{ "wsrep_apply_oooe",                "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_apply_oool",                "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_causal_reads",              "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_commit_oooe",               "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_commit_oool",               "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_flow_control_recv",         "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_flow_control_sent",         "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_flow_control_paused",       "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_local_bf_aborts",           "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_local_cert_failures",       "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_local_commits",             "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_local_replays",             "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_received",                  "operations",   DS_TYPE_DERIVE },
+		{ "wsrep_replicated",                "operations",   DS_TYPE_DERIVE },
+
+		{ "wsrep_received_bytes",            "total_bytes",  DS_TYPE_DERIVE },
+		{ "wsrep_replicated_bytes",          "total_bytes",  DS_TYPE_DERIVE },
+
+		{ "wsrep_apply_window",              "window_size",  DS_TYPE_GAUGE },
```

Comment:
it's bytes indeed, but i'm pretty fine about using total_bytes easier to use when graphing , just to be sure that's a sum not a number :)

hoping to be clear